doc/api/http.md

```diff
@@ -1808,13 +1808,22 @@ A collection of all the standard HTTP response status codes, and the
 short description of each. For example, `http.STATUS_CODES[404] === 'Not
 Found'`.
 
-## http.createServer([options][, requestListener])
+## http.createServer([requestListener])
 <!-- YAML
 added: v0.1.13
-changes:
-  - version: v9.6.0
-    pr-url: https://github.com/nodejs/node/pull/15752
-    description: The `options` argument is supported now.
+-->
+- `requestListener` {Function}
+
+* Returns: {http.Server}
+
+Returns a new instance of [`http.Server`][].
+
+The `requestListener` is a function which is automatically
+added to the [`'request'`][] event.
+
+## http.createServer([options][, requestListener])
```

Comment:
On 8.x-staging, you'll need to remove this stuff too. This function signature doesn't work on 8.x, which is what this doc change is fixing.